docs/src/pages/components/toggle-button/ToggleButtons.js

```diff
@@ -38,17 +38,22 @@ export default function ToggleButtons() {
     <Grid container spacing={2}>
       <Grid item sm={12} md={6}>
         <div className={classes.toggleContainer}>
-          <ToggleButtonGroup value={alignment} exclusive onChange={handleAlignment}>
-            <ToggleButton value="left">
+          <ToggleButtonGroup
+            value={alignment}
+            exclusive
+            onChange={handleAlignment}
+            aria-label="Text alignment"
+          >
+            <ToggleButton value="left" aria-label="Left aligned">
```

Comment:
Screen readers can differentiate capitalized letters (as far as I have searched the topic), you should notice that all the aria, values in the codebase are lowercased, at least, I have done an effort in this direction recently, I have tried. I would propose the following: 

```suggestion
            <ToggleButton value="left" aria-label="left aligned">
```

Reply:
> Screen readers can't differentiate capitalized letters

Nope, it was purely stylistic. I'll make them consistent.

Reply:
> Screen readers can't differentiate capitalized letters

Do you have a source for that? That would be new to me and quite a big missing feature for narration. For example `React` and `react` should be read differently.

Reply:
I could only find weak signals, https://stackoverflow.com/questions/49930658/should-aria-labels-be-capitalized, test with VoiceOver, Bootstrap docs approach.

Reply:
Well that guy doesn't have a source either apart from his job title.

NVDA has various settings for capital letters:
* extra sound (default: false)
* Speak "Capital" (default: false)
* raise voice by x% (default: +30%)
![nvda-capital-settings](https://user-images.githubusercontent.com/12292047/64160307-06442680-ce3c-11e9-96ee-8462720bfd3e.png)

But that currently only applies to single letters. Different voices depending on word casing is still open https://github.com/nvaccess/nvda/issues/3286.

So we should be mindful of capitalization.